src/BlazorUI/Bit.BlazorUI/Components/Surfaces/ScrollablePane/BitScrollablePane.scss

```
@@ -33,3 +78,56 @@
+// rendering above off the screen everywhere it actually works.
+@supports not selector(::-webkit-scrollbar) {
+    .bit-scp-mod {
+        scrollbar-width: thin;
+        scrollbar-color: $clr-fg-ter transparent;
+    }
+}
+
+// The fade over the edges that still have content beyond them. It is a mask rather than a pair of
+// gradient overlays, so it works over any background and needs no element of its own inside a pane whose
+// content the consumer owns; and it is two masks composited together, so the two axes fade independently.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)